src/sentry/apidocs/api_pagination_allowlist_dont_modify.py

```diff
@@ -0,0 +1,140 @@
+"""
+    This list is tracking old api endpoints that don't correctly implement pagination.
+    The goal is to eventually find owners for all and shrink this list.
+    DO NOT ADD ANY NEW APIS
+"""
+API_PAGINATION_ALLOWLIST_DONT_MODIFY = [
```

Comment:
nit: could you alphabetize this list? There's a nifty [extension](https://marketplace.visualstudio.com/items?itemName=Tyriar.sort-lines) that does it for you